### PR TITLE
Rebranded the Explore Section, replaced download link and added discontinued to the correct functions

### PR DESCRIPTION
**Pull request by Akira-Kuru**
- Replaced high fidelity in the Explore Section to Project Athena. 
- Replaced the download link with the correct one. 
- Added the discontinued functions with a (discontinued) beside the title.

### CODE REVIEW

File: docs/source/explore/shop.rst [hunk 1]
```diff
@@ -2,35 +2,39 @@
...
 
-If you want to buy items in High Fidelity, head on over to the Marketplace, where you can purchase more than 300 items built by digital artists and creators from around this world. High Fidelity uses their own cryptocurrency, High Fidelity Coin (HFC) to manage your transactions. 
+If you want to buy items in Project Athena, head on over to the Marketplace, where you can purchase more than 300 items built by digital artists and creators from around this world. Project Athena uses their own cryptocurrency, Project Athena Coin (HFC) to manage your transactions. 
 
 .. contents:: On This Page
     :depth: 2
 
 -----------------------------
-Buy High Fidelity Coin
+Buy Project Athena Coin (DISCONTINUED)
```
Comment (kasenvr) [inline docs/source/explore/shop.rst]:
```suggestion
Buy High Fidelity Coin (DISCONTINUED)
```

File: docs/source/explore/shop.rst [hunk 2]
```diff
@@ -2,35 +2,39 @@
...
 
-Currently, you can buy High Fidelity Coins (HFC) using `Ethereum <https://www.ethereum.org>`_, a blockchain app that trades ETH (Ether). The Bank of High Fidelity manages HFC and we gradually increase the number of coins in circulation as the economy grows. 100 HFC is equal to 1 USD.
+This feature was discontinued by High Fidelity and will either be reworked or removed in the future.
+
+Currently, you can buy Project Athena Coins (HFC) using `Ethereum <https://www.ethereum.org>`_, a blockchain app that trades ETH (Ether). The Bank of Project Athena manages HFC and we gradually increase the number of coins in circulation as the economy grows. 100 HFC is equal to 1 USD.
 
 To get HFC: 
 
-1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of High Fidelity.
+1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of Project Athena.
```
Comment (kasenvr) [inline docs/source/explore/shop.rst]:
```suggestion
1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of High Fidelity.
```

File: docs/source/explore/shop.rst [hunk 3]
```diff
@@ -2,35 +2,39 @@
...
 2. At the time of your appointment, visit the `TradingRoom <https://hifi.place/TradingRoom>`_ domain.
-3. We will provide you with a QR code that you can use to send ETH to the Bank of High Fidelity. 
+3. We will provide you with a QR code that you can use to send ETH to the Bank of Project Athena.
```
Comment (kasenvr) [inline docs/source/explore/shop.rst]:
```suggestion
3. We will provide you with a QR code that you can use to send ETH to the Bank of High Fidelity. 
```

File: docs/source/explore/shop.rst [hunk 4]
```diff
@@ -2,35 +2,39 @@
...
 
-1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of High Fidelity.
+1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of Project Athena.
 2. At the time of your appointment, visit the `TradingRoom <https://hifi.place/TradingRoom>`_ domain.
-3. We will provide you with a QR code that you can use to send ETH to the Bank of High Fidelity. 
+3. We will provide you with a QR code that you can use to send ETH to the Bank of Project Athena. 
 4. Once we receive the ETH amount you sent, we will send you the appropriate amount of HFC to your account based on the current exchange rate. This can take up to one business day. 
 
-You can also receive HFC from gifts from your friends or as prizes at events in High Fidelity.
+You can also receive HFC from gifts from your friends or as prizes at events in Project Athena.
```
Comment (kasenvr) [inline docs/source/explore/shop.rst]:
```suggestion
You can also receive HFC from gifts from your friends or as prizes at events in High Fidelity.
```

File: docs/source/explore/shop.rst [hunk 5]
```diff
@@ -2,35 +2,39 @@
...
 
-As you acquire more HFC through Marketplace sales, prizes, or gifts, you may cash out your HFC for USD. The minimum amount of HFC that you cash out is $25 (or 2,500 HFC), and the maximum cashout value is $2000 (or 200,000 HFC) per calendar month. High Fidelity may, at its discretion, issue the occasional exception to the maximum cashout amount.
+This feature was discontinued by High Fidelity and will either be reworked or removed in the future.
+
+As you acquire more HFC through Marketplace sales, prizes, or gifts, you may cash out your HFC for USD. The minimum amount of HFC that you cash out is $25 (or 2,500 HFC), and the maximum cashout value is $2000 (or 200,000 HFC) per calendar month. Project Athena may, at its discretion, issue the occasional exception to the maximum cashout amount.
 
 To cash out HFC: 
 
-1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of High Fidelity.
+1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of Project Athena.
```
Comment (kasenvr) [inline docs/source/explore/shop.rst]:
```suggestion
1. `Book an appointment <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_ at the Bank of High Fidelity.
```

File: docs/source/explore/shop.rst (hunk 1 above)
Comment (kasenvr) [reply docs/source/explore/shop.rst]: This was very specifically named High Fidelity Coin and was a High Fidelity project. We'll make Athena Coin later. 😜

Comment (Akira-Kuru) [reply docs/source/explore/shop.rst]: Fixed it.

File: docs/source/explore/shop.rst (hunk 3 above)
Comment (Akira-Kuru) [reply docs/source/explore/shop.rst]: Fixed

File: .gitignore [hunk 6]
```diff
@@ -1 +1,3 @@
-*/build
\ No newline at end of file
+*/build
+
+.idea/
```
Comment (kasenvr) [inline .gitignore]: Everything looks good, but one final question, what's the reason for this?

Comment (Akira-Kuru) [reply .gitignore]: I use Phpstorm thats why the ignore .idea and */build because i don’t want to commit the built version.

Comment (kasenvr) [reply .gitignore]: I gotcha, I wasn't aware, never used that IDE. Thanks! :)

**Pull request merged**